tests/utils/skipif.php

```diff
@@ -39,6 +39,86 @@ function skip_if_not_replica_set()
     is_replica_set(URI) or exit('skip topology is not a replica set');
 }
 
+/**
+ * Skips the test if the topology has no arbiter.
+ */
+function skip_if_no_arbiter()
+{
+    try {
+        $primary = get_primary_server(URI);
+    } catch (ConnectionException $e) {
+        exit('skip server is not accessible: ' . $e->getMessage());
```

Comment:
Should this message also say "primary server" for consistency?

Reply:
Will fix before merging.